kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -42,6 +42,12 @@ data:
           # When the setting is set to false, only the kube events with !normal event type will be collected
           enabled = false
           # When this is enabled (enabled = true), all kube events including normal events will be collected
+       #[log_collection_settings.schema]
```

Comment:
@austonli , we are working on the release. let me know if you want to take this change as part of the current agent release.